lib/3scale/backend/logging.rb

```diff
@@ -11,11 +11,11 @@ def self.included(base)
         enable! on: base
       end
 
-      def self.enable!(on:, with: [], as: :logger)
-        logger = if with.empty?
+      def self.enable!(on:, as: :logger, with_args: [], with_opts: {})
+        logger = if with_args.empty? && with_opts.empty?
```

Comment:
previously setting anything but empty `CONFIG_WORKERS_LOG_FILE` resulted in a crash

Now I believe old behavior is retained but one can also set a log file